tests/test_management.py

```diff
@@ -266,33 +266,33 @@ def test_transmit_course_metadata_task_success(
             '' + uuid + '/course/' + course_run_ids[0] + '/enroll/", '
             '"videoUrl": ""}, {"authors": [], "categoryTags": [], "contentId": "'+course_run_ids[1]+'", '
             '"costType": "Paid", "description": "edX Demonstration Course", "difficulty": "", "duration": 0, '
             '"format": "Instructor", "imageUrl": "", "institution": "", "language": "en", "publishDate": "2013-02-05", '
-            '"title": "edX Demonstration Course", "url": "' + settings.LMS_ROOT_URL + '/enterprise/'
+            '"title": "edX Demonstration Course (Starts: February 2013)", "url": "' + settings.LMS_ROOT_URL + '/enterprise/'
             '' + uuid + '/course/' + course_run_ids[1] + '/enroll/", '
             '"videoUrl": ""}], "orgCode": "Degreed Company", "providerCode": "DEGREED"}'
         )
         sapsf_expected_dump = (
             '{"ocnCourses": [{"content": [{"contentID": "' + course_run_ids[0] + '", '
-            '"contentTitle": "edX Demonstration Course", "launchType": 3, "launchURL": '
+            '"contentTitle": "edX Demonstration Course (Starts: February 2013)", "launchType": 3, "launchURL": '
             '"' + settings.LMS_ROOT_URL + '/enterprise/' + uuid + '/course/' + course_run_ids[0] + '/enroll/'
             '", "mobileEnabled": false, "providerID": "SAP"}], "courseID": "' + course_run_ids[0] + '"'
             ', "description": [{"locale": "English", "value": "edX Demonstration Course"}], '
             '"price": [], "providerID": "SAP", "revisionNumber": 1, "schedule": '
             '[{"active": true, "endDate": 2147483647000, "startDate": 1360040400000}], '
             '"status": "ACTIVE", "thumbnailURI": "", "title": [{"locale": "English", '
-            '"value": "edX Demonstration Course"}]}, {"content": [{"contentID": '
-            '"' + course_run_ids[1] + '", "contentTitle": "edX Demonstration Course", "launchType": 3, '
+            '"value": "edX Demonstration Course (Starts: February 2013)"}]}, {"content": [{"contentID": '
+            '"' + course_run_ids[1] + '", "contentTitle": "edX Demonstration Course (Starts: February 2013)", "launchType": 3, '
             '"launchURL": "' + settings.LMS_ROOT_URL + '/enterprise/' + uuid + '/course/'
             '' + course_run_ids[1] + '/enroll/", "mobileEnabled": false, "providerID": "SAP"}], '
             '"courseID": "' + course_run_ids[1] + '", "description": [{"locale": "English", '
             '"value": "edX Demonstration Course"}], "price": [], "providerID": "SAP", '
             '"revisionNumber": 1, "schedule": [{"active": true, "endDate": 2147483647000, '
             '"startDate": 1360040400000}], "status": "ACTIVE", "thumbnailURI": "", '
-            '"title": [{"locale": "English", "value": "edX Demonstration Course"}]}]}'
+            '"title": [{"locale": "English", "value": "edX Demonstration Course (Starts: February 2013)"}]}]}'
```

Comment:
You could also probably make the course we are using by default for these tests self-paced so you don't have to make these changes.